scheduler/src/cook/mesos.clj

```diff
@@ -140,6 +143,25 @@
                               (apply max))]
     utilization))
 
+(defn make-trigger-chans
+  [rebalancer-config
+   {:keys [timeout-interval-minutes]
+    :or {timeout-interval-minutes 10}
+    :as task-constraints}]
+  (let [match-trigger-chan (async/chan (async/sliding-buffer 1))]
+    ;; Use a normal channel for match-trigger-chan because other events besides chimes will go on channel
+    (async/pipe (chime-ch (periodic/periodic-seq (time/now) (time/seconds 1)))
+                match-trigger-chan)
+    {:rebalancer-trigger-chan (chime-ch (periodic/periodic-seq (time/now)
+                                                               (time/seconds (:interval-seconds rebalancer-config))))
+     :match-trigger-chan match-trigger-chan
+     :rank-trigger-chan (chime-ch (periodic/periodic-seq (time/now) (time/seconds 5)))
```

Comment:
Yes, but can it wait for a different PR? I'd like to keep this PR small and.. it is no worse than it was =)